src/services/kpis/kpis.svc.coffee

```diff
@@ -42,10 +56,12 @@ angular
     @load = (force=false) ->
       deferred = $q.defer()
 
-      if _.isEmpty(_self.config.ssoSessionId) || force
+      if _.isEmpty(_self.getSsoSessionId()) || force
         ImpacMainSvc.loadUserData(force).then(
           (mainConfig) ->
-            _self.config.ssoSessionId = mainConfig.sso_session
+            _self.config.userData.id = mainConfig.id
+            _self.config.userData.orgUids = _.map(mainConfig.organizations, (o)-> o.uid)
```

Comment:
@cesar-tonnoir `@getAlerts`, I only need the user id.. but we may also need the user orgUids later for org wide alerts